cmplibrary/src/main/java/com/sourcepoint/cmplibrary/SpConsentLibImpl.kt

```diff
@@ -297,6 +297,8 @@ internal class SpConsentLibImpl(
 
         val gdprGroupPmId = campaignManager.getGroupId(campaignType)
 
+        pLogger.i("loadPm - isOtt: ", isOtt.toString())
```

Comment:
Is this log desirable? Just asking 👀

Reply:
yes, pLogger in prod is an empty object, in debug allows us to test if the `isOtt ` has the right value